apps/dog-food/Main.hs

```diff
@@ -36,8 +36,9 @@ cfg :: AgentConfig
 cfg =
   AgentConfig
     { tcpPort = undefined, -- TODO maybe take it out of config
-      tbqSize = 16,
+      rsaKeySize = 256,
```

Comment:
maybe specify bytes as ```(2048 `div` 8)```? This would probably be more readable as crypto primitives are usually sized in bits

Reply:
yes - good idea - updated everywhere, I was also confused initially